src/Layout/Presentation/presentation-utilities.ts

```diff
@@ -1,15 +1,35 @@
 import { PartyResult, DistrictResult, LevelingSeat, PartyRestQuotients } from "../../computation";
 import { roundNumber } from "../../utilities/number";
 
+function filterPartiesWithoutSeats(partyResults: PartyResult[], nationalResults?: PartyResult[]) {
+    let filteredResults = [...partyResults];
+
+    if (nationalResults) {
+        filteredResults = filteredResults.filter((party) => {
+            const nationalParty = nationalResults.find((nationalParty) => nationalParty.partyCode === party.partyCode);
+            if (!nationalParty) {
+                return false;
+            } else {
+                return nationalParty.totalSeats > 0;
+            }
+        });
+    } else {
+        filteredResults = filteredResults.filter((party) => party.totalSeats > 0);
+    }
```

Comment:
Why do we use "else"-s here? Wouldn't it be better to specifically state what we're checking for?